stdlib/public/Distributed/DistributedActorSystem.swift

```diff
@@ -12,28 +12,267 @@
 import Swift
 import _Concurrency
 
-/// A distributed actor system is what enables all functionality of distributed actors.
+/// A distributed actor system underpins and implements all functionality of distributed actors.
 ///
-/// Distributed actors must always be associated with a concrete distributed actor system,
-/// and do so by declaring a `typealias ActorSystem = ...`, or by having a module wide
-/// `typealias DefaultDistributedActorSystem` declared which then applies to all distributed
-/// actors that do not declare a specific type alias in their bodies.
+/// A ``DistributedActor`` is always initialized in association with some concrete actor system.
+/// And that actor system instance is then used to manage the identity of the actor, as well as
+/// handle all remote interactions of the distributed actor.
+///
+/// ## Using a DistributedActorSystem library
+///
+/// From a library user's perspective (e.g. someone using a `ClusterSystem` or `SampleWebSocketActorSystem`),
+/// the basic use of a distributed actor system is fairly opaque.
+///
+/// Any distributed actor must declare what actor system it is able to operate with. This is done either by a
+/// `typealias ActorSystem` in the body of such `distributed actor` declaration, or a module-wide global
+/// `typealias DefaultDistributedActorSystem`. Refer to the ``DistributedActor`` documentation to learn more
+/// about the tradeoffs of these approaches.
+///
+/// Once an actor has declared the system it is able to work with, such system instance must provided
+/// at initialization time, in order for the system to be able to take over the actor's identity management.
+///
+/// For example, a simple distributed actor may look like this:
+///
+/// ```swift
+/// distributed actor Greeter {
+///   init(name: String, actorSystem: ActorSystem) {
+///     self.name = name
+///     self.actorSystem = actorSystem // required (!) initialization of implicit actorSystem property
+///   }
+/// }
+/// ```
+/// Notice that every distributed actor initializer must initialize the synthesized ``DistributedActor/actorSystem``.
+/// This property is later used for identity management and other remote interactions of the actor.
+/// For more details refer to ``DistributedActor`` which explains more about declaring distributed actors.
+///
+/// For more details about how the specific actor system implementation deals with remote message transports,
+/// and serialization, please refer to the specific system's documentation.
+///
+/// > Note: For example, you may refer to the [Swift Distributed Actors cluster library](https://github.com/apple/swift-distributed-actors/) documentation,
+/// > which is one example of such feature complete distributed actor system implementation:
```

Comment:
`.` instead of `:` or is there something else you'd want to add there?

Reply:
. is good, thanks